x-pack/plugins/security_solution/server/endpoint/services/artifacts/manifest_manager/manifest_manager.ts

```diff
@@ -586,17 +663,30 @@ export class ManifestManager {
   private async listEndpointPolicies(page: number) {
     return this.packagePolicyService.list(this.savedObjectsClient, {
       page,
-      perPage: 20,
+      perPage: 100,
       kuery: 'ingest-package-policies.package.name:endpoint',
     });
   }
 
-  private async listEndpointPolicyIds(page: number) {
-    return this.packagePolicyService.listIds(this.savedObjectsClient, {
-      page,
-      perPage: 20,
-      kuery: 'ingest-package-policies.package.name:endpoint',
-    });
+  private async listEndpointPolicyIds() {
+    const allPolicyIds: string[] = [];
+
+    let paging = true;
+    let page = 1;
+
+    while (paging) {
+      const { items, total } = await this.packagePolicyService.listIds(this.savedObjectsClient, {
+        page,
+        perPage: 100,
+        kuery: 'ingest-package-policies.package.name:endpoint',
+      });
+
+      allPolicyIds.push(...items);
+
+      paging = (page - 1) * 20 + items.length < total;
```

Comment:
you forgot to update `20` here to `100` (or 1000 if you take on my suggestion fo increasing it even further)

Reply:
https://github.com/elastic/kibana/pull/160387/commits/07f122e199a58e825182d9fc2c26fe4c27f18809